server/src/main/java/io/crate/expression/predicate/IsNullPredicate.java

```diff
@@ -90,6 +90,12 @@ public Symbol normalizeSymbol(Function symbol, TransactionContext txnCtx, NodeCo
         if (arg instanceof Input<?> input) {
             return Literal.of(input.value() == null);
         }
+        if (arg instanceof Reference ref
+            // allow system columns for further error processing, for example see testSelectWhereVersionIsNullPredicate().
```

Comment:
thx for the comment, is there an easy way to validate the usage of `_version` before we do the normalization? I see for example in `RelationAnalyzer#visitQuerySpecification()` that the `WhereClauseValidator` is called before the symbol normalization, but maybe there are other cases?